.github/workflows/ci.yml

```diff
@@ -287,7 +287,6 @@ jobs:
 
           echo "CARGO_TARGET_AARCH64_UNKNOWN_LINUX_MUSL_RUNNER=qemu-aarch64" >> $GITHUB_ENV
           echo "/musl/bin" >> $GITHUB_PATH
-
```

Comment:
Remove this commit please.

Reply:
? It’s needed for musl builds for arm on x86